java/code/src/com/redhat/rhn/manager/system/VirtualizationActionMap.java

```diff
@@ -55,49 +55,56 @@ private void setup() {
         startMap.put("stopped", ActionFactory.TYPE_VIRTUALIZATION_START);
         startMap.put("crashed", ActionFactory.TYPE_VIRTUALIZATION_START);
         startMap.put("paused", ActionFactory.TYPE_VIRTUALIZATION_RESUME);
+        startMap.put("unknown", ActionFactory.TYPE_VIRTUALIZATION_START);
```

Comment:
What is the `unknown` key used for?

Reply:
the VM may be in an unknown state in some weird circumstances (mostly a libvirt-events or libvirt bug)....